ddl/ddl_test.go

```diff
@@ -78,13 +78,30 @@ func (ts *testSuite) TestT(c *C) {
 	tbStmt = statement("create table t2 (a int unique not null)").(*stmts.CreateTableStmt)
 	err = sessionctx.GetDomain(ctx).DDL().CreateTable(ctx, tbIdent2, tbStmt.Cols, tbStmt.Constraints)
 	c.Assert(err, IsNil)
+	tb, err := sessionctx.GetDomain(ctx).InfoSchema().TableByName(tbIdent2.Schema, tbIdent2.Name)
+	c.Assert(err, IsNil)
+	c.Assert(tb, NotNil)
+	_, err = tb.AddRecord(ctx, []interface{}{1})
+	_, err = tb.AddRecord(ctx, []interface{}{2})
+	c.Assert(err, IsNil)
+	alterStmt := statement(`alter table t2 add b enum("bbb") first`).(*stmts.AlterTableStmt)
```

Comment:
Should add a test that after u add a new column, u can select the updated value.